tests/gatsby-starter-default/src/templates/default.js

```diff
@@ -8,45 +8,46 @@ import Layout from '../components/Layout';
 import NewsItem from '../components/NewsItem';
 
 const componentFor = data => {
+  const nodes = query => (query ? query['edges'] : []).map(edge => edge.node);
   if (data) {
-    if (data.ploneCollection) {
+    if (data['ploneCollection']) {
       return (
         <Folder
```

Comment:
If we have a collection component, why do we still use Folder component here to handle ploneCollections?

Reply:
Correct. I'll fix the template to use it.